Sources/PrimerSDK/Classes/User Interface/Root/PrimerUniversalCheckoutViewController.swift

```diff
@@ -23,6 +23,7 @@ internal class PrimerUniversalCheckoutViewController: PrimerFormViewController {
     private var onClientSessionActionUpdateCompletion: ((Error?) -> Void)?
     private var singleUsePaymentMethod: PrimerPaymentMethodTokenData?
     private var resumePaymentId: String?
+    private var cardButtonViewModel: CardButtonViewModel!
```

Comment:
Can we avoid this implicitly unwrapped optional? It seems we have no guarantees it will be present (viewDidLoad can bypass setting it) so it would be better to stay safe

Reply:
Updated, thanks!